### PR TITLE
Add tooling for easy-access to job details

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Enable spyglass for easy-access to job details

**Related issue(s)**

See #57

### CODE REVIEW

**Comment by michal-hudy** [comment]
@suleymanakbas91 Please don't change production cluster configuration before PR is merged...

File: prow/cluster/05-gcsweb.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## Source is https://github.com/kubernetes/k8s.io/tree/master/gcsweb.k8s.io
```
Comment (michal-hudy) [inline prow/cluster/05-gcsweb.yaml]: I think it should be deployed before TLS ingress

File: prow/gubernator/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.coverage
```
Comment (michal-hudy) [inline prow/gubernator/.gitignore]: How about creating fork on `kyma-incubator` with Gubernator changes? Right now is it hard to determine what changes did you make. Also, as you said it is only temporary.

Take also in mind that we are cloning this repository in almost every job.

Comment (suleymanakbas91) [reply prow/gubernator/.gitignore]: This is a very good idea! I'll update the PR in this way.

**Comment by aszecowka** [comment]
Is it requirement that bucket where we store logs needs to be publicly available? If so, we need to document it.

**Comment by michal-hudy** [comment]
It is possible to display other logs like `clone-log.txt` ?

**Comment by suleymanakbas91** [comment]
> Is it possible to display other logs like `clone-log.txt` ?

When there is an issue in `clone-log.txt`, it is shown under `Build Log`. But when it is successful I think there is no way to see it from the view page. For that, we can keep track of the improvements in Spyglass.

**Pull request merged**